examples/posenet/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html lang = "en">
+  <head> 
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title> PoseNet Image Demo</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <style>
+      #gui{
+        position: relative;
+        top: 100px;
+      }
+      .main{
+        position: absolute;
+        left: 50px;
+        top: 100px;
+        right: 0px;
+      }
+      .col{
+        position: relative;
+        top: 20px;
+      }
+      .Single,.Multiple {
+        display: inline-block;
+      }
+    </style>
+  </head>
+
+  <body>
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark" style="width:100%">
+      <span class="navbar-brand mb-0 h1">WebML PoseNet Demo</span>
+        <div class="btn-group">
+          <select onchange = "drawSingleandMulti();" id = "backend" class="btn btn-success dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <option value = "webml">WebML</option>
```

Comment:
Use buttons, please refer to https://github.com/intel/webml-polyfill/blob/master/examples/mobilenet/index.html#L30